clowdapp.yaml

```diff
@@ -10,6 +10,8 @@ objects:
     name: ros-ocp-backend
   spec:
     envName: ${ENV_NAME}
+    testing:
+      iqePlugin: ros_ocp
```

Comment:
s/ros_ocp/ros-ocp

https://consoledot.pages.redhat.com/docs/dev/operating-your-app/testing-iqe/testing.html#_clowdapp_settings 

The name of the plugin you use here should be the "short name" of your plugin, and should match the name of your plugin’s image tag in quay.io